stocks/Authentication/LoginViewController.swift

```diff
@@ -36,19 +53,36 @@ class LoginViewController: ViewController {
         return v
     }()
     
-    lazy var content = ScrollableStackView(views: [email, password], spacing: 10)
+    lazy var content = ScrollableStackView(views: [email, password], spacing: 40 )
+    
     
     override func viewDidLoad() {
         super.viewDidLoad()
+      
+        
+        view.addSubview(image)
         view.addSubview(content)
         view.addSubview(logIn)
+        
+        image.snp.makeConstraints { (make) in
+            make.size.edges.equalTo(view.safeAreaLayoutGuide)
```

Comment:
make.edges.equalTo(view) - we want the background to cover the full screen, not just the safeAreaLayoutGuide.
we want the image to go behind the navigation bar, all the way to the top of the screen. the navigation bar should be transparent, and the cancel button probably white or green.